### PR TITLE
Mostly cleanup.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

**Comment by dkorolev** [comment]
@mzhurovich Don't Panic! It's just the cleanup.

File: Blocks/HTTP/api.h [hunk 1]
```diff
@@ -57,15 +57,15 @@ SOFTWARE.
...
-#elif defined(BRICKS_APPLE)
+#elif defined(CURRENT_APPLE)
 #include "impl/apple_client.h"
 #include "impl/posix_server.h"
 typedef blocks::HTTPClientApple HTTP_CLIENT;
-#elif defined(BRICKS_JAVA) || defined(BRICKS_ANDROID)
+#elif defined(CURRENT_JAVA) || defined(CURRENT_ANDROID)
 #include "impl/java_client.h"
 #include "impl/posix_server.h"
 typedef blocks::aloha::HTTPClientPlatformWrapper HTTP_CLIENT;
```
Comment (mzhurovich) [inline Blocks/HTTP/api.h]: Hmm, we still have `aloha` somewhere :-/

**Comment by mzhurovich** [comment]
@dkorolev Okay :)

File: Blocks/HTTP/api.h (hunk 1 above)
Comment (dkorolev) [inline Blocks/HTTP/api.h]: Good eye! This piece is used by Android/Java code by Alex though, and it's not clear to me as of now whether we should keep or retire it.

**Pull request merged**